### PR TITLE
Announcing pecas dashboards for freckle

### PR DESCRIPTION
**Pull request by etagwerker**
Hey @mauro-oto, 

This PR adds an article announcing http://ombulabs.github.io/pecas/

Please check it out.

Thanks!

### CODE REVIEW

**Comment by mauro-oto** [comment]
@etagwerker could you check the conflicts and update this PR? Thanks!

File: _posts/2016-05-30-introducing-pecas.markdown [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: "Introducing Pecas: Dashboards for Freckle"
+date: 2016-05-16 12:31:00
```
Comment (mauro-oto) [inline _posts/2016-05-30-introducing-pecas.markdown]: Could you update this date? Thanks!

**Comment by mauro-oto** [comment]
The blog post looks good! :+1:

**Comment by etagwerker** [comment]
@mauro-oto done!

**Pull request merged**